Batch Convert.py

```diff
@@ -0,0 +1,46 @@
+## Instructions (Python 3.5+ and running on Japanese/Korean Locale)
+# 1. Edit the 'inputs' and 'output' directory which contains the library of BMS files
+#    you want to convert and where to output.
+# 2. Run this script.
+# 3. In the same directory you placed the Python script, search for convert.bat (or whatever
+#    you set for 'output_list', and place it in the folder with SM2BMConverter.exe, then run the .bat file.
+
+## Example:
+# BM2SMConverter.exe -I "D:/Insane BMS/2016-08-21 proposals/love_this_moment_ogg/love_this_moment_n.bme" -O "D:/BMS SM Convert/love_this_moment_ogg love_this_moment_n" -K S1234567
+
+## NOTES:
+# This script assists in converting a very large collection of BMS files.
+# For example, if you have the Insane BMS folder, you can convert just
+# about every possible file. This script does not search through subdirectories
+# however.
+#
+# Folders are named after their parent folder (song) and filename without the extension.
```

Comment:
This may cause collisions easily.
Say, you want to batch convert a folder with two charts named 7K.bml and 7K.bms. The final names will be the same so overwritten.